cpp/daal/include/algorithms/pca/pca_types.h

```diff
@@ -672,6 +672,8 @@ class DAAL_EXPORT BaseBatchParameter : public daal::algorithms::Parameter
     DAAL_UINT64 resultsToCompute; /*!< 64 bit integer flag that indicates the results to compute */
     size_t nComponents;           /*!< number of components for reduced implementation */
     bool isDeterministic;         /*!< sign flip if required */
+    bool doScale;                 /*!< scaling if required */
+    bool isCorrelation;           /*!< correlation is provided */
```

Comment:
Why do you need this parameter? It was not here before.

Reply:
Its necessary for align compute calls between svd and cov method. Because previous compute took isCorrelation as an input parameter